ui/app/machines/replication-machine.js

```diff
@@ -19,7 +19,6 @@ export default {
     },
     complete: {
       onEntry: ['completeFeature'],
-      on: { RESET: 'idle' },
```

Comment:
Do we still have reset for replication or no?

Reply:
We do. I removed RESET from all of the complete steps because completeFeature basically dismantles the machine so RESET would never be called.